### ISSUE TITLE
How would I add a class specifically for the questions of panel type

### ISSUE THREAD

**Issue by john1625b** [surveyjs/surveyjs#611]
### Are you requesting a feature, reporting a bug or ask a question?
Question

### What is the current behavior?
With the custom css classes, I am unable to add a class to questions of panel type(like class="panel-container"). The panel type questions I am referring to:
![screen shot 2017-08-30 at 5 54 50 pm](https://user-images.githubusercontent.com/17265085/29896870-a46148a2-8dac-11e7-8643-11b538d83fcf.png)

Does survey.js currently have a method of solving this issue?
Thanks

**Comment by andrewtelnov** [comment]
@john1625b I have added to classes: panel.title and panel.container

Thank you,
Andrew

**Comment by john1625b** [comment]
@andrewtelnov Thank you so much for adding this feature. How would I go about getting these changes? I have upgraded my node modules so now my survey-react version is ^0.12.23. I added the panel object to my custom styles like so:
`panel: {title: "panel-title", container: "panel-container"},`
But when I inspect the panel element on my project, I do not see the relevant classes showing up. Has the commit not been released to npm yet?

Thanks
